loops-dane-alex.js

```diff
@@ -0,0 +1,92 @@
+// Write a for loop that logs each number from 1 - 20.
+
+// for (let i = 1; i < 21; i++) {
+//     console.log(i)
+// }
+
+// Write a for loop that logs the result of each number from 1 - 20 tripled.
+
+// for (let i = 1; i < 21; i++) {
+//     console.log(i * 3)
+// }
+
+// Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+
+// for (let i = 1; i < 21; i++) {
+//     if (i % 2 === 0) {
+//         console.log(i)
+//     } else {
+//         console.log("ODD")
+//     }
+// }
+
+
+// Looping over an array. Consider this variable:
+// var nums = [3, 57, -9, 20, 67]
+
+// Create a loop that will log the highest number from the array. Expected output --> 67
+var nums = [3, 57, -9, 20, 67]
+var largestnum
```

Comment:
The beauty of coding is that there are so many ways to obtain the desired output. Nevertheless, there are also best practices. 
1. When using conditional statements, use only one `if`  condition and as many `else if` conditions as necessary. 
2. Line 29 is an undefined variable if you were to console.log() the largestnum it would be undefined. I recommend assigning a value initially, such as 
var largestnum = nums[0]
By declaring a variable and assigning a value, you can now remove line 31.
3. Indentation - helps with readability and separate the concerns. For instance, place condition and executable code on separate lines.

Reply:
The question I asked myself here was "What if we are looking for the largest number with all negative numbers in the array?". If we were looking for the largest number of negative numbers and initialized largestnum to 0, then the output of largestnum would be 0 which would not be correct. I know it's not really relevant to this particular problem, but how would we go about solving this problem that were the case?